spec/models/case/overturned_ico/foi_spec.rb

```diff
@@ -108,14 +108,13 @@
           it 'errors' do
             new_case.received_date = 41.days.ago
             expect(new_case).not_to be_valid
-            expect(new_case.errors[:received_date]).to eq ['decision received from ICO must be within 40 days of today']
-          end
+            expect(new_case.errors[:received_date]).to eq ["of decision from ICO must be within 40 days of today"]
```

Comment:
FYI, `end` has been removed
```suggestion
            expect(new_case.errors[:received_date]).to eq ["of decision from ICO must be within 40 days of today"]
          end
```